Script/AtomicEditor/hostExtensions/languageExtensions/TypscriptLanguageExtension.ts

```diff
@@ -337,6 +341,42 @@ export default class TypescriptLanguageExtension implements Editor.HostExtension
         WebView.WebBrowserHost.setGlobalStringProperty("TypeScriptLanguageExtension", "tsConfig", JSON.stringify(tsConfig));
     }
 
+    generateExternalProject ()
+    {
+      var projectPath : string = ToolCore.toolSystem.project.projectPath;
+
+      // Create TypeScript folder in project root
+      var tsPath : string = projectPath + "TypeScript/";
```

Comment:
addTrailingSlash will add an internal slash, which is always "/", it will also normalize the path to internal representation:

https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Source/Atomic/IO/FileSystem.cpp#L1017

GetNativePath will convert a path string to use native "\" on Windows, though this is seldom necessary: https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Source/Atomic/IO/FileSystem.cpp#L1049